src/backend/common/CommandBuffer.cpp

```diff
@@ -564,6 +570,20 @@ namespace backend {
                     }
                     break;
 
+                case Command::SetStencilReference:
+                    {
+                        SetStencilReferenceCmd* cmd = iterator.NextCommand<SetStencilReferenceCmd>();
+                        if (lastPipeline->IsCompute()) {
```

Comment:
Compute workloads are done outside of renderpasses and graphics inside so you don't need to check for lastPipeline->IsCompute.